src/test/java/com/google/api/codegen/testdata/nodejs_test_library.baseline

```diff
@@ -0,0 +1,719 @@
+============== file: test/test.js ==============
+* Copyright 2016 Google Inc. All rights reserved.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+
+var assert = require('assert');
+var service = require('../src/').v1();
+
+describe('LibraryServiceClient', function() {
+  describe('createShelf', function() {
```

Comment:
We will have at least two tests for each method (happy vs unhappy cases). Ideally I would prefer the output has the hierachy which looks like:

ServiceName:
     methodName:
          Test methodName without error --- PASS
          Test methodName with error --- PASS